src/config/constants.ts

```diff
@@ -267,7 +267,8 @@ export const DOCUMENTATION = {
     DEVTRON_UPGRADE: `${DOCUMENTATION_HOME_PAGE}/v/v0.5/getting-started/upgrade`,
     APP_METRICS: `${DOCUMENTATION_HOME_PAGE}/v/v0.5/usage/applications/app-details/app-metrics`,
     EXTERNAL_SECRET: `${DOCUMENTATION_HOME_PAGE}/v/v0.5/usage/applications/creating-application/secrets#external-secrets`,
-    BLOB_STORAGE: `${DOCUMENTATION_HOME_PAGE}/v/v0.5/getting-started/install/installation-configuration#configuration-of-blob-storage`
+    BLOB_STORAGE: `${DOCUMENTATION_HOME_PAGE}/v/v0.5/getting-started/install/installation-configuration#configuration-of-blob-storage`,
+    CLUSTER_AND_ENVIRONMENTS: `${DOCUMENTATION_HOME_PAGE}/getting-started/global-configurations/cluster-and-environments`,
```

Comment:
it seems similar to GLOBAL_CONFIG_CLUSTER. Please verify and remove

Reply:
reused older link